core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.html

```diff
@@ -117,31 +117,47 @@
+          <br>
+          <div class="outline-Finalized-container" *ngIf="oldOutline">
             Outline Finalized
             <i *ngIf="!outlineIsFinalized" (click)="finalizeOutline()" class="far fa-square md-18 e2e-test-finalize-outline" aria-label="Finalize outline"></i>
             <i *ngIf="outlineIsFinalized" (click)="unfinalizeOutline()" class="fas fa-check-square md-18" aria-label="Unfinalize outline"></i>
-            <div tabindex="-1" (blur)="toggleChapterOutlineButtons()" (click)="toggleChapterOutlineButtons()">
+          </div>
+          <div *ngIf="!chapterOutlineIsShown">
+            <div class="outline-content-container" *ngIf="!oldOutlineToReadOnly">
+              <input class="chapter-outline-editor-content e2e-test-node-outline-editor-content e2e-test-rte"
+                     readonly
+                     (click)="chapterOutlineIsShown=true"
+                     placeholder="Open chapter outline editor.">
+              <label class="fas fa-pencil-alt oppia-icon" (click)="chapterOutlineIsShown=true"></label>
+            </div>
+            <div class="outline-content-container" *ngIf="oldOutlineToReadOnly" >
+              <button class="chapter-outline-editor-content e2e-test-node-outline-editor-content e2e-test-rte"
+                      (click)="chapterOutlineIsShown=true">{{oldOutlineToReadOnly}}
+              </button>
+              <label class="fas fa-pencil-alt oppia-icon" (click)="chapterOutlineIsShown=true"></label>
+            </div>
+          </div>
+          <br>
+          <div class="chapter-card-inputs oppia-mobile-collapsible-card-content" *ngIf="chapterOutlineIsShown">
+            <div tabindex="-1" >
               <schema-based-editor id="storyNodeOutline"
                                    [schema]="OUTLINE_SCHEMA"
                                    [localValue]="editableOutline"
-                                   (localValueChange)="updateLocalEditableOutline($event)">
+                                   (localValueChange)="updateLocalEditableOutline($event);"
+                                   (click)="chapterOutlineButtonsAreShown=true; ">
               </schema-based-editor>
               <div class="save-button-container" *ngIf="chapterOutlineButtonsAreShown">
-                <button class="btn btn-default">Cancel</button>
+                <button type="button"
+                        class="btn btn-default e2e-test-node-outline-cancel-button"
+                        (click)="onCancelButtonClicked();">
+                  Cancel
+                </button>
                 <button type="button"
                         class="btn btn-success save-button e2e-test-node-outline-save-button"
                         [disabled]="!isOutlineModified(editableOutline)"
-                        (click)="updateOutline(editableOutline)">
+                        (click)="updateOutline(editableOutline); onSaveButtonClicked();">
```

Comment:
Can updateOutline(editableOutline) be included in onSaveButtonClicked() (and removed from here), or will this break something else?

Reply:
@hiramatsuyuusuke did you see this comment?

Reply:
@hiramatsuyuusuke Just flagging this again. Could you please respond? Thanks.